vos/new-commands-README.md

```diff
@@ -0,0 +1,99 @@
+## vostools 3.1.0
```

Comment:
Couldn't this just be in the _actual_ `README`?

Reply:
There's already a README in root. So I've moved it as README.md to commands and linked it from the README in root. Good point - thanks.